packages/DataGrid/tests/DataGrid.cypress.js

```diff
@@ -87,7 +87,7 @@ describe("<DataGrid />", () => {
       });
   });
 
-  it("should render ColumnIndicator", () => {
```

Comment:
does this keep failing?

Reply:
is strange that this start happening with this PR, I'm just curious 🤔

Reply:
what the heck, i re-enabled those tests ... maybe the push didnt work although i'm sure i saw the commit here a few minutes ago....  one sec.